pkg/pgmodel/ingestor/insert.go

```diff
@@ -381,18 +381,18 @@ func doInsert(conn pgxconn.PgxConn, reqs ...copyRequest) (err error) {
 		return err
 	}
 
-	for i, numRows := range numRowsPerInsert {
+	var affectedMetrics uint64
+	for _, numRows := range numRowsPerInsert {
 		ct, err := results.Exec()
 		if err != nil {
 			return err
 		}
 		if int64(numRows) != ct.RowsAffected() {
-			log.Warn("msg", "duplicate data in sample", "table", reqs[i].table, "duplicate_count", int64(numRows)-ct.RowsAffected(), "row_count", numRows)
-			metrics.DuplicateSamples.Add(float64(int64(numRows) - ct.RowsAffected()))
-			metrics.DuplicateWrites.Inc()
+			affectedMetrics++
+			registerDuplicates(int64(numRows) - ct.RowsAffected())
 		}
 	}
+	reportDuplicates(affectedMetrics)
```

Comment:
I am actually not sure how to interpret `affectedMetrics` This is the number of metrics that had duplicates in a batch, but I'm really not sure what this tells me especially since I dont get the sum, just one number.

Reply:
It represents the number of metrics that have got duplicate samples. Do you think this needs to be changed to "sum of all duplicate rows count?"